drafts/2019-12-24-lets-get-to-know-kabanero-pipelines.adoc

```diff
@@ -17,4 +17,73 @@ Each Kabanero collection contains  a set of default tasks and pipelines that are
 
 == Process of running your first Kabanero Pipeline
 
-While there are two ways of running your Kabanero Pipeline, link:https://github.com/tektoncd/experimental/blob/master/webhooks-extension/docs/GettingStarted.md[using Automated Webhooks] is the preferred one. link:https://github.com/kabanero-io/draft-guide-working-with-pipelines/blob/master/pipelines.adoc[This guide] takes you step by step on creating and running your Kabanero Pipeline. link:https://github.com/tektoncd/dashboard[Tekton Dashboard] makes it easy to configure your pipelines as well as see them in action with detailed logs. For more news and releases on Kabanero Pipelines, follow our link:https://twitter.com/Kabaneroio[social media channels]. 
+While there are two ways of running your Kabanero Pipeline, link:https://github.com/tektoncd/experimental/blob/master/webhooks-extension/docs/GettingStarted.md[using Automated Webhooks] is the preferred one. 
+
+To bein with, you'll need to have link:https://github.com/kabanero-io/kabanero-foundation[Kabanero foundation installed] on Red Hat Origin Community Distribution of Kubernetes (OKD) or OpenShift Container Platform (OCP) cluster. link:https://github.com/kabanero-io/kabanero-pipelines[create a persistent volume], create secrets to pull from git repo and push to docker registry link:https://github.com/kabanero-io/kabanero-pipelines[following this link] and finally, execute pipelines using linl:https://github.com/tektoncd/experimental/blob/master/webhooks-extension/docs/GettingStarted.md[Tekton Dashboard Webhook Extension].
```

Comment:
a few suggestions:
- I would move word "installed" out of the hotlink
- The hotlink "create a persistent volume" should start with a capital C as it's beginning of new sentence in the paragraph. 
- fix typo, "linl" should be "link for last hotlink.